### DISCUSSION TITLE
the assumption of capacity factor in electricity apartment

### DISCUSSION THREAD

**Discussion by mengdanni** [JGCRI/gcam-core#349] [Q&A]
In GCAM v6, what is the assumption of capacity factor in electricity apartment, I CAN'T FIND IT.

**Comment by pkyle** [comment]
Capacity factors can be queried with the following query (which works for GCAM-USA states as well as standard model regions, despite the different structures used in each). This query can be copied to the clipboard, and pasted into the "Queries" window of the model interface, with a model output database open. Or it can be added to output/Main_queries.xml.
`
			<supplyDemandQuery title="elec capacity factors">
			    <axis1 name="technology">technology</axis1>
			    <axis2 name="Year">technology</axis2>
			    <xPath buildList="true" dataName="capacity-factor" group="false" sumAll="false">*[@type='sector' and (@name='electricity' or @name='base load generation' or 
               @name='intermediate generation' or @name='subpeak generation' or @name='peak generation' or @name='elect_td_bld')]//*[@type='subsector']//
              *[@type='technology' and not(@name='elect_td_bld')]/capacity-factor/node()</xPath>
			    <comments/>
			</supplyDemandQuery>
`
